backend/src/main/java/gov/cdc/usds/simplereport/api/deviceType/DeviceTypeMutationResolver.java

```diff
@@ -0,0 +1,39 @@
+package gov.cdc.usds.simplereport.api.organization;
+
+import java.util.List;
+import java.util.UUID;
+
+import graphql.kickstart.tools.GraphQLMutationResolver;
+import org.springframework.stereotype.Component;
+
+import gov.cdc.usds.simplereport.service.DeviceTypeService;
+import gov.cdc.usds.simplereport.db.model.DeviceType;
+
+@Component
+public class DeviceTypeMutationResolver implements GraphQLMutationResolver {
+
+    private final DeviceTypeService _dts;
+
+    public DeviceTypeMutationResolver(DeviceTypeService dts) {
+        _dts = dts;
+    }
+
+    public DeviceType createDeviceType(
+        String name,
+        String manufacturer,
+        String model,
+        String loincCode
+    ) throws Exception {
```

Comment:
I have to assume that one of our static analysis tools is going to yell about the too-broad exception here. :-)